package.json

```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "csv-parse": "^4.16.3",
-    "eslint": "^8.2.0",
+    "eslint": "^7.0.0",
```

Comment:
prevents warnings about peer dependencies from our own eslint modules, new enough for our needs